src/components/table/LightTable.tsx

```diff
@@ -379,11 +382,14 @@ function LightTable({
                 disableTouchRipple: true,
                 disableFocusRipple: true,
                 disableRipple: true,
+                disabled: infinitePaging
+                  ? rows.length < rowsPerPage
+                  : page * rowsPerPage + rows.length >= filteredCount,
```

Comment:
Since the paging works for the case we have infinitePaging set to false, what would you think about just setting this for the case that infinitePaging is enabled? 
```suggestion
                disabled: infinitePaging && rows.length < rowsPerPage
```
Otherwise, I think there is something off with this argument: `page * rowsPerPage + rows.length >= filteredCount`. Shouldn't the inequality sign be flipped?

Reply:
this is for "disabled", so if there are more results than there are results, then it should be disabled. I think i could change it to == instead of >=?

Reply:
I cant set the disabled for just infinite paging, i think? And if the attribute is set, I need to have it update for !infinitepaging situations

Reply:
> I cant set the disabled for just infinite paging, i think? And if the attribute is set, I need to have it update for !infinitepaging situations

Ah, you're totally right. 

> this is for "disabled", so if there are more results than there are results, then it should be disabled. I think i could change it to == instead of >=?

If there are more results than we can display on the page then I think we want disabled=false so that it will continue to page, right? Double negatives are fun haha

Reply:
Yeah, haha, Im sorry about that! XD

it's saying `disable if (page * rowsPerPage + rows.length >= filteredCount)`, which should mean there are no more results! I think it's right, from clicking around in the app!

Reply:
Thanks for chatting with me about this offline!

Here's my takeaway from this conversation:
I was hung up on the two different conditions (especially since the equality was flipped), but I think I understand now--
-> When infinitePaging=true, we don't know the total count, so we work with what we have, which works for every case EXCEPT number of rows == rows per page
-> When infinitePagine=false, we have the totalRow count and therefore can handle the edge case where the number of rows == rows per page, but we've hit the end of the results.